source/guides/network/supernode-enrollment.rst

```diff
@@ -84,54 +94,86 @@ After following that guide, you need to **install a monitoring agent** and **ann
 
 1. **Download the node monitoring agent**:
 
-   `Download the Linux binary <https://symbol-node-reward.s3-eu-west-1.amazonaws.com/packages/1.0.0/agent_binary/agent-linux.bin>`__ and make it executable:
+   `Download the Linux binary <https://symbol-node-reward.s3-eu-west-1.amazonaws.com/packages/2.0.0/agent_binary/agent-linux.bin>`__ and make it executable:
```

Comment:
I think DH wants to keep the version 2.0.0 for the mainnet release so we are going to release 1.1.0 for the testnet I think.

Reply:
OK, we won't publish this until we know for sure.